src/test/java/movieinformation/mapper/MovieInformationMapperTest.java

```diff
@@ -102,4 +99,14 @@ class MovieInformationMapperTest {
         assertThat(insertMovie).isNotEmpty();
 
     }
+
+    //PATCHのDBテスト
```

Comment:
PATCHはHTTPの言葉なので
```suggestion
    // UPDATEのDBテスト
```
で良いと思います。

Reply:
PATCH自体が更新を意味する英単語なので間違いではないですけどね。

Reply:
こちら修正しておきます！